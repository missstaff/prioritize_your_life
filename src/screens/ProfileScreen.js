import { Text, View, useColorScheme } from "react-native";
import { useTheme } from "@react-navigation/native";
import { getFireApp } from "../../getFireApp";


const ProfileScreen = () => {
   
    const firebase = getFireApp();
    const scheme = useColorScheme();
    let { colors } = useTheme();
    
    return (
        <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
            <Text>Profile Screen</Text>
        </View>
    );
};

export default ProfileScreen;