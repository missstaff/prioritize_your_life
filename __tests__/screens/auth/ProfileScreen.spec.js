import renderer from "react-test-renderer";
import ProfileScreen from "../../../src/screens/ProfileScreen";



describe("ProfileScreen", () => {
  it("ProfileScreen should render correctly", () => {
    const tree = renderer.create(<ProfileScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  });
  afterEach(() => {
    jest.clearAllMocks();
  });
});