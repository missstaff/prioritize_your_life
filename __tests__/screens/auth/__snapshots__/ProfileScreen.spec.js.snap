// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen ProfileScreen should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Profile Screen
  </Text>
</View>
`;
